Makefile

```diff
@@ -35,6 +35,11 @@ bin/example-app: check-go-version
 release-binary:
 	@go build -o _output/bin/dex -v -ldflags $(LD_FLAGS) $(REPO_PATH)/cmd/dex
 
+.PHONY: revendor
+revendor:
```

Comment:
This is kinda already addressed in our documentation.

> Dex uses glide and glide-vc to manage its vendor directory. A recent version of these are preferred but dex doesn't require any bleeding edge features. Either install these tools using go get or take an opportunity to update to a more recent version.

If we ever need something that was added in a specific version of glide we can add a script similar to `scripts/check-go-version`, but I don't really want to write a bunch of `sed` commands right now :/